cert_issuer/config.py

```diff
@@ -33,48 +33,49 @@ def configure_logger():
 def add_arguments(p):
     p.add('-c', '--my-config', required=False, env_var='CONFIG_FILE',
           is_config_file=True, help='config file path')
-    p.add_argument('--issuing_address', required=True, help='issuing address')
-    p.add_argument('--usb_name', required=True, help='usb path to key_file')
+    p.add_argument('--issuing_address', required=True, help='issuing address', env_var='ISSUING_ADDRESS')
+    p.add_argument('--usb_name', required=True, help='usb path to key_file', env_var='USB_NAME')
     p.add_argument('--key_file', required=True,
-                   help='name of file on USB containing private key')
+                   help='name of file on USB containing private key', env_var='KEY_FILE')
     p.add_argument('--unsigned_certificates_dir', default=os.path.join(DATA_PATH, 'unsigned_certificates'),
-                   help='Default path to data directory storing unsigned certs')
+                   help='Default path to data directory storing unsigned certs', env_var='UNSIGNED_CERTIFICATES_DIR')
     p.add_argument('--signed_certificates_dir', default=os.path.join(DATA_PATH, 'signed_certificates'),
-                   help='Default path to data directory storing signed certs')
+                   help='Default path to data directory storing signed certs', env_var='SIGNED_CERTIFICATES_DIR')
     p.add_argument('--blockchain_certificates_dir', default=os.path.join(DATA_PATH, 'blockchain_certificates'),
-                   help='Default path to data directory storing blockchain certs')
+                   help='Default path to data directory storing blockchain certs', env_var='BLOCKCHAIN_CERTIFICATES_DIR')
     p.add_argument('--work_dir', default=WORK_PATH,
-                   help='Default path to work directory, storing intermediate outputs. This gets deleted in between runs.')
-    p.add_argument('--max_retry', default=10, type=int, help='Maximum attempts to retry transaction on failure')
+                   help='Default path to work directory, storing intermediate outputs. This gets deleted in between runs.', env_var='WORK_DIR')
+    p.add_argument('--max_retry', default=10, type=int, help='Maximum attempts to retry transaction on failure', env_var='MAX_RETRY')
     p.add_argument('--chain', default='bitcoin_regtest',
-                   help='Which chain to use. Default is bitcoin_regtest (which is how the docker container is configured). Other options are bitcoin_testnet bitcoin_mainnet, mockchain, ethereum_mainnet, ethereum_ropsten')
+                   help=('Which chain to use. Default is bitcoin_regtest (which is how the docker container is configured). Other options are'
```

Comment:
Might need a space after the `are` at the end:  `Other options are'`

Reply:
oops, nice catch, I've pushed a fix